web/views.py

```
@@ -6928,6 +6931,171 @@ def all_study_groups(request):
+
+    context = {
+        "submission": submission,
+        "status_choices": submission.get_status_choices(),
+        "is_student": is_student,
+        "is_reviewer": is_reviewer,
+    }
+    return render(request, "web/work/submission_detail.html", context)
+
+
+# Add this helper view to serve files directly (important for development)
+def serve_work_file(request: HttpRequest, file_path: str) -> HttpResponse:
```

Ruff (0.8.2)

Unused function argument: `request`

(ARG001)